activemq-broker/src/main/java/org/apache/activemq/broker/jmx/DestinationView.java

```diff
@@ -620,4 +621,67 @@ public long getNetworkDequeues() {
         return destination.getDestinationStatistics().getNetworkDequeues().getCount();
     }
 
+    @Override
+    public boolean isAdvancedMessageStatisticsEnabled() {
+        return destination.isAdvancedMessageStatisticsEnabled();
+    }
+
+    @Override
+    public void setAdvancedMessageStatisticsEnabled(boolean advancedMessageStatisticsEnabled) {
+        destination.setAdvancedMessageStatisticsEnabled(advancedMessageStatisticsEnabled);
+    }
+
+    @Override
+    public long getEnqueuedMessageBrokerInTime() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getEnqueuedMessageBrokerInTime().getValue() : 0l);
+    }
+
+    @Override
+    public String getEnqueuedMessageClientId() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getEnqueuedMessageClientID().getValue() : null);
+    }
+
+    @Override
+    public String getEnqueuedMessageId() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getEnqueuedMessageID().getValue() : null);
+    }
+
+    @Override
+    public long getEnqueuedMessageTimestamp() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getEnqueuedMessageTimestamp().getValue() : 0l);
+    }
+
+    @Override
+    public long getDequeuedMessageBrokerInTime() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getDequeuedMessageBrokerInTime().getValue() : 0l);
+    }
+
+    @Override
+    public long getDequeuedMessageBrokerOutTime() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getDequeuedMessageBrokerOutTime().getValue() : 0l);
+    }
+
+    @Override
+    public String getDequeuedMessageClientId() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getDequeuedMessageClientID().getValue() : null);
+    }
+
+    @Override
+    public String getDequeuedMessageId() {
+        MessageFlowStats tmpMessageFlowStats = destination.getDestinationStatistics().getMessageFlowStats();
+        return (tmpMessageFlowStats != null ? tmpMessageFlowStats.getDequeuedMessageID().getValue() : null);
+    }
+
+    @Override
+    public long getDequeuedMessageTimestamp() {
```

Comment:
I was thinking all of these getters could be a bit cleaner and less copy/paste with using lambdas. IE.

```java
@Override
public long getDequeuedMessageTimestamp() {
    return getMessageFlowStat(MessageFlowStats::getDequeuedMessageTimestamp, 0L);
}

private <T> T getMessageFlowStat(Function<MessageFlowStats, UnsampledStatistic<T>> f, T defVal) {
    final var stats = destination.getDestinationStatistics().getMessageFlowStats();
    return stats != null ? f.apply(stats).getValue() : defVal;
}
```